example/src/components/buttons/StandardButton.tsx

```diff
@@ -0,0 +1,102 @@
+import { AdditionalColors, BrandColors, TextColors } from '@colors';
+import { Typo } from '@components/Typo';
+import React, { ReactNode } from 'react';
+import {
+  StyleSheet,
+  View,
+  Pressable,
+  GestureResponderEvent,
+} from 'react-native';
+
+const StandardButtonStyles = StyleSheet.create({
+  common: {
+    width: '100%',
+    height: 56,
+    borderRadius: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  primary: {
+    backgroundColor: BrandColors.darkBlue100,
+  },
+  danger: {
+    backgroundColor: AdditionalColors.red100,
+  },
+  info: {
```

Comment:
maybe it'd be better to name it "secondary"?

Reply:
Renamed.